pkg/printers/internalversion/printers.go

```diff
@@ -517,6 +521,29 @@ func AddHandlers(h printers.PrintHandler) {
 	}
 	h.TableHandler(validatingWebhookColumnDefinitions, printValidatingWebhook)
 	h.TableHandler(validatingWebhookColumnDefinitions, printValidatingWebhookList)
+
+	flowSchemaColumnDefinitions := []metav1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "PriorityLevel", Type: "string", Description: flowcontrolv1alpha1.PriorityLevelConfigurationReference{}.SwaggerDoc()["name"]},
+		{Name: "MatchingPrecedence", Type: "string", Description: flowcontrolv1alpha1.FlowSchemaSpec{}.SwaggerDoc()["matchingPrecedence"]},
+		{Name: "DistinguisherMethod", Type: "string", Description: flowcontrolv1alpha1.FlowSchemaSpec{}.SwaggerDoc()["distinguisherMethod"]},
+		{Name: "MatchesAll", Type: "bolean", Description: "matches all requests"},
```

Comment:
dont think we need a `MATCHESALL` column, the mandatory name `catch-all` will be sufficient..

Reply:
It is logically possible, and actually happens in our latest thoughts about what should be in the suggested config, that other schemas besides "catch-all" match every request.

Good catch about IsDangling, I will add that.

Reply:
added

Reply:
My original thought was that it is OK if this is only accurate for the schemas we supply and the ones people write following our lead.  However, on second thought, I think not.  If the column appears then it should be computed accurately for all possible schemas.  I think that is possible, but more trouble than it is worth at this stage of development.  So I have removed that column.

Reply:
@jktomer on third thought, I like the recognition that was here.  Can you cite an example of a FlowSchema that matches every request but would not have been recognized as such by the code that was here?

Reply:
For easy reference, the recognition code is at https://github.com/MikeSpreitzer/kubernetes/blob/4fbbdcdac78080d4611e6e27229803b6df248fbb/pkg/printers/internalversion/printers.go#L2276-L2310

Reply:
For example, one that catches a group of which all users actually authorized to send any requests are members. Without looking at the actual ACLs you won't know that it catches everything, but no request not caught by that flow schema will ever make it to the queue.

Reply:
Yes, if other semantics are brought to bear then there can be examples.  I was thinking of a "matches all" column whose definition is purely in terms of the syntax of this API.  I think that's meaningful enough to be useful.

Reply:
Looks like MatchesAll got removed in the current PR and I think I agree with that, at least for a first take. If people ask for it, we can reopen the discussion?